routing-daemon/lib/openshift/routing/models/nginx.rb

```diff
@@ -197,8 +160,7 @@ def add_ssl pool_name, alias_str, ssl_cert, private_key
         @logger.debug("Adding SSL configuration for alias #{alias_str} for pool #{pool_name}")
         File.open(fname, "w") {|file| file.puts new_text }
       rescue Errno::ENOENT
-        # Nothing to do; if server.conf doesn't exist, then that means
-        # there are no routes, so we should just return the empty array.
+        # Nothing to do;
```

Comment:
I think we can keep the comment and just s/there are no routes/the alias has not been created yet/.